src/helm/config/model_metadata.yaml

```diff
@@ -1184,6 +1184,23 @@ models:
     # TODO(#1828): Upgrade to FULL_FUNCTIONALITY_TEXT_MODEL_TAG
     tags: [TEXT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG]
 
+  - name: meta/llama-3-8b
+    display_name: Llama 3 (8B)
+    description: TBD  # TODO: Fill in description
```

Comment:
For speed of getting this up, do we want to merge without this TODO done?

Reply:
Added the description.